examples/manipulating_visualizing/plot_prob_atlas.py

```diff
@@ -0,0 +1,47 @@
+"""
+Plotting 4D Atlas maps or regions onto the anatomical image
+===========================================================
+
+We show here how to plot the atlas maps or statistical maps which are
+4D using a three different display types, for example using
```

Comment:
Since we have bug in ```display.add_overlay``` when plotting Harvard Oxford Atlas, can I change the atlas name to MSDL ?